aws/graph-explorer
# Upgrade prettier, eslint, husky, pnpm

Issue #, if available:

Description of changes:

* Fix issue running Prettier where a syntax error is incorrectly reported
* Update Prettier to latest
* Update Husky to latest
* Update ESLint and plugins to latest
* Update PNPM to latest using CorePack
* Add `format:check` script to run Prettier without writing the files
* Add `lint:check` script to run ESLint without writing the files

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
